var/spack/repos/builtin/packages/libsplash/package.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Libsplash(Package):
+    """
+    libSplash aims at developing a HDF5-based I/O library for HPC simulations.
+    It is created as an easy-to-use frontend for the standard HDF5 library
+    with support for MPI processes in a cluster environment. While the
+    standard HDF5 library provides detailed low-level control, libSplash
+    simplifies tasks commonly found in large-scale HPC simulations, such as
+    iterative computations and MPI distributed processes.
+    """
+
+    homepage = "https://github.com/ComputationalRadiationPhysics/libSplash"
+    url      = "https://github.com/ComputationalRadiationPhysics/libSplash/archive/v1.4.0.tar.gz"
+
+    version('1.4.0', '2de37bcef6fafa1960391bf44b1b50e0')
+    version('1.3.1', '524580ba088d97253d03b4611772f37c')
+    version('1.2.4', '3fccb314293d22966beb7afd83b746d0')
+
+    variant('mpi', default=True,
+            description='Enable parallel I/O (one-file aggregation) support')
+
+    depends_on('cmake', type='build')
+    depends_on('hdf5@1.8.6:', when='~mpi')
+    depends_on('hdf5@1.8.6:+mpi', when='+mpi')
```

Comment:
Thank you!

I was wondering because `spack graph libsplash` did not show `hdf5` but `spack graph --concretize libsplash` does in the list...

will update with your suggestion, thank you both!